utils/build-script-impl

```diff
@@ -2012,7 +2012,6 @@ for host in "${ALL_HOSTS[@]}"; do
                     -DCLANG_TOOL_ARCMT_TEST_BUILD=NO
                     -DCLANG_TOOL_C_ARCMT_TEST_BUILD=NO
                     -DCLANG_TOOL_C_INDEX_TEST_BUILD=NO
-                    -DCLANG_TOOL_DRIVER_BUILD=NO
```

Comment:
Since we already *have* `BUILD_TOOLCHAIN_ONLY`, though, it's more important to not affect how that works if you're *not* cross-compiling.